cypress/e2e/marc-authority/marc-bibliographic-unlink-fields.cy.js

```diff
@@ -0,0 +1,171 @@
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+
+describe('MARC -> MARC Bibliographic', () => {
+  const testData = {
+    tag100: '100',
+    tag010: '010',
+    tag700: '700',
+    authority100FieldValue: 'C365598 Chin, Staceyann, 1972-',
+    authority010FieldValue: 'n2008052404',
+    authority700FieldValue: 'C365598 Woodson, Jacqueline',
+    successMsg:
+      'This record has successfully saved and is in process. Changes may not appear immediately.',
+    accordion: 'Contributor',
+  };
+  let createdInstanceID;
+  const marcFiles = [
+    {
+      marc: 'marcFileForC365598_1.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcFileForC365598_2.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcFileForC365598_3.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      numOfRecords: 1,
+    },
+  ];
+  const linkedField = {
+    rowIndex: 11,
+    tag: '100',
+    secondBox: '1',
+    thirdBox: '\\',
+    content: '$a C365598 Chin, Staceyann, $d 1972-',
+    eSubfield: '$e Author $e Narrator',
+    zeroSubfield: '$0 id.loc.gov/authorities/names/n2008052404365598',
+    seventhBox: '$1 http://viaf.org/viaf/24074052',
+  };
+  const unlinkedField = {
+    rowIndex: 11,
+    tag: '100',
+    indicator0: '1',
+    indicator1: '\\',
+    content:
+      '$a C365598 Chin, Staceyann, $d 1972- $e Author $e Narrator $0 id.loc.gov/authorities/names/n2008052404365598 $1 http://viaf.org/viaf/24074052',
+  };
+  const contributors = {
+    firstName: 'C365598 Chin, Staceyann, 1972-',
+    secondName: 'Woodson, Jacqueline',
+  };
+  const createdAuthorityIDs = [];
+
+  before(() => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiQuickMarcQuickMarcEditorDuplicate.gui,
+      Permissions.uiQuickMarcQuickMarcAuthorityLinkUnlink.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+      Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+      Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
+      Permissions.moduleDataImportEnabled.gui,
+      Permissions.uiCanLinkUnlinkAuthorityRecordsToBibRecords.gui,
+      Permissions.dataExportEnableApp.gui,
+    ]).then((createdUserProperties) => {
+      testData.userProperties = createdUserProperties;
+
+      cy.login(testData.userProperties.username, testData.userProperties.password, {
+        path: TopMenu.dataImportPath,
+        waiter: DataImport.waitLoading,
+      });
+      marcFiles.forEach((marcFile) => {
+        cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(
```

Comment:
I'd propose not to make multiple logins but perform login one time and navigate to start page in the loop. Please look at the next example: 
```JS
      cy.loginAsAdmin().then(() => {
        marcFiles.forEach((marcFile) => {
          cy.visit(TopMenu.dataImportPath);
          DataImport.waitLoading();

          DataImport.uploadFile(marcFile.marc, marcFile.fileName);
          JobProfiles.waitLoadingList();
          JobProfiles.searchJobProfileForImport(marcFile.jobProfileToRun);
          JobProfiles.runImportFile();
          JobProfiles.waitFileIsImported(marcFile.fileName);
          Logs.checkStatusOfJobProfile('Completed');
          Logs.openFileDetails(marcFile.fileName);
          for (let i = 0; i < marcFile.numOfRecords; i++) {
            Logs.getCreatedItemsID(i).then((link) => {
              createdAuthorityIDs.push(link.split('/')[5]);
            });
          }
        });
      });
```

Reply:
resolved